### DISCUSSION TITLE
Problem with mesh creation using SimNIBS 4.1

### DISCUSSION THREAD

**Discussion by umne** [simnibs/simnibs#455] [7.Creating head models] [answered]
Hello everyone, 
I'm approaching for the first time SimNIBS 4.1 using the GUI, and I started by creating some head meshes from T1 and T2 weighted images starting from the windows command prompt and the 'charm' function. 
For the majority of images things went smooth and I obtained a pretty clear image (see first figure). But for some other subjects the mesh was really smooth and didn't change as I tried to repeat the mesh using only the T1 image (see second figure). 

Has anyone had any issue like this before? Many thanks in advance for your insights :)

![image (1)](https://github.com/user-attachments/assets/64a47f30-8479-4b48-bad9-bfcb03aced44)
![image (2)](https://github.com/user-attachments/assets/e3a87541-88f2-47e7-9ba1-2b1efdfe0056)

**Comment by jduemose** [comment, marked as answer]
Hi did you check whether the segmentation is reasonable in general? I mean, is it only the gray matter that looks bad? If so, you can run freesurfer first and then use the cortical surfaces (white and pial) from freesurfer with charm. To point charm to the freesurfer result, use the option called `--fs-dir` (when you call charm) and specify the path to the freesurfer run of the subject, so

```
charm [...] --fs-dir /path/to/fs/run
```

Best,
Jesper

**Reply by umne** [reply]
Thank you for your response!
It probably is an issue with the automatic segmentation. I tried selecting and reassigning the CSF portion of the tissue to another Physical Element but I cannot manage from Gmesh to click on a sufficient portion of CSF to reassign it. I'm trying with freesurfer segmentation and let you know
